AstronomyObjC/AstronomyObjC/Networking/CDGFetchPhoto/CDGFetchPhoto.m

```diff
@@ -0,0 +1,86 @@
+//
+//  CDGFetchPhoto.m
+//  AstronomyObjC
+//
+//  Created by Chris Gonzales on 4/21/20.
+//  Copyright © 2020 Chris Gonzales. All rights reserved.
+//
+
+#import "CDGFetchPhoto.h"
+#import "CDGPhoto.h"
+#import "AstronomyObjC-Swift.h"
+
+@interface CDGFetchPhoto ()
+
+@property (nonatomic, strong, readwrite, nullable) UIImage *image;
+@property (nonatomic, strong) NSURLSessionDataTask *dataTask;
+
+@property BOOL internalIsExecuting;
+@property BOOL internalIsFinished;
+
+@end
+
+@implementation CDGFetchPhoto { BOOL _isFinished;
+BOOL _isExecuting;
+}
```

Comment:
Don't mix and match these approaches.

Don't use ivars: `_isFinished` if you're going to use the `internalIsExecuting` or you could introduce a logic error.